### DISCUSSION TITLE
Spotify no sound output after update to 20210125

### DISCUSSION THREAD

**Discussion by Chboss74** [rern/rAudio#142] [Fixed Issues]
First of all thank you for the great software. 
I used Volumio many years ago and gave Rune a try a few weeks ago with great success.
System: Raspi 4 / USB DAC SMSL M100

Initially I had no issues streaming from Spotify to Rune.
Today I updated to V20210125 and Spotify connects and shows the cover art and counter running but no sound output.

Playing local files from SSD and Streaming from Airplay works normally.
I tried the obvious rebooting Rune and restarting Spotify without success.

**Comment by Roland-B** [comment]
@ Rern - Same here.

# systemctl status spotifyd

● spotifyd.service - A spotify playing daemon
     Loaded: loaded (/usr/lib/systemd/system/spotifyd.service; enabled; vendor preset: disabled)
    Drop-In: /etc/systemd/system/spotifyd.service.d
             └─override.conf
     Active: active (running) since Tue 2021-01-26 20:32:47 GMT; 20s ago
       Docs: https://github.com/Spotifyd/spotifyd
    Process: 6710 ExecStartPost=/srv/http/bash/cmd.sh nicespotify (code=exited, status=0/SUCCESS)
   Main PID: 6709 (spotifyd)
      Tasks: 4 (limit: 4915)
     CGroup: /system.slice/spotifyd.service
             └─6709 /usr/bin/spotifyd --no-daemon

Jan 26 20:32:59 rAudio taskset[6709]: ALSA lib pcm.c:2660:(snd_pcm_open_noupdate) Unknown PCM "default:CARD=E30"
Jan 26 20:32:59 rAudio taskset[6709]: Alsa error PCM open ALSA function 'snd_pcm_open' failed with error 'ENOENT: No such file or directory'
Jan 26 20:32:59 rAudio taskset[6709]: Could not start audio: Alsa error: PCM open failed
Jan 26 20:32:59 rAudio taskset[6709]: ALSA lib pcm.c:2660:(snd_pcm_open_noupdate) Unknown PCM "default:CARD=E30"
Jan 26 20:32:59 rAudio taskset[6709]: Alsa error PCM open ALSA function 'snd_pcm_open' failed with error 'ENOENT: No such file or directory'
Jan 26 20:32:59 rAudio taskset[6709]: Could not start audio: Alsa error: PCM open failed
Jan 26 20:32:59 rAudio taskset[6709]: Running "/srv/http/bash/spotifyd.sh" using "/bin/bash" with environment variables {"TRACK_ID": "16Yd41R1G5KOG4PzsCfZzu", "PLAYER_EVENT": "start"}
Jan 26 20:33:04 rAudio taskset[6709]: ALSA lib pcm.c:2660:(snd_pcm_open_noupdate) Unknown PCM "default:CARD=E30"
Jan 26 20:33:04 rAudio taskset[6709]: Alsa error PCM open ALSA function 'snd_pcm_open' failed with error 'ENOENT: No such file or directory'

RolandB.

**Comment by Chboss74** [comment]
OK just updated to 20210129 and Spotify works again!
Thank you for the quick fix.
